src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
You don't need `flex` styles to the card
Each block element will be displayed on the new line by default